beta/src/components/MDX/Challenges/Navigation.tsx

```diff
@@ -11,19 +11,19 @@ const debounce = require('debounce');
 export function Navigation({
   challenges,
   handleChange,
-  activeChallenge,
+  currentChallenge,
   isRecipes,
 }: {
   challenges: ChallengeContents[];
   handleChange: (id: string) => void;
-  activeChallenge: string;
+  currentChallenge?: ChallengeContents;
```

Comment:
Why is this nullable? Shouldn't there always be a current challenge?

Reply:
Yes there _**should**_ be a current challenge... However, because the `currentChallenge` variable comes from a `find()` TypeScript can't guarantee its existence. 
A fix/hack would be to place a `!` on the `find()` however, I wasn't sure that is allowed in the coding style guide (and haven't spent a lot of time with TypeScript).
If you're okay with that, I can make the appropriate changes.

Reply:
Seems ok.

Reply:
We can also throw if not found.